### PR TITLE
I don't wanna hear it

### PR DESCRIPTION
**Pull request by eriklsy**
:D
sorry

### CODE REVIEW

File: frontend/app/index.tsx [hunk 1]
```diff
@@ -1,9 +1,8 @@
-import React, { useState } from 'react';
+import React, { useState } from "react";
```
Comment (jaykbpark) [inline frontend/app/index.tsx:1]: bruh

File: frontend/components/ui/input-field.tsx [hunk 2]
```diff
@@ -1,41 +1,20 @@
...
-  inputStyle,
-}) => {
+const InputField = () => {
   return (
-    <View style={[styles.container, containerStyle]}>
-      <TextInput
-        style={[styles.input, inputStyle]}
-        secureTextEntry={type === "password"}
-      />
+    <View>
```
Comment (jaykbpark) [inline frontend/components/ui/input-field.tsx]: why is it wrapped with a view?  @eriklsy

**Review by jaykbpark — APPROVED** [review]
lgtm -- please check why it needs to be wrapped in view

File: frontend/components/ui/input-field.tsx (hunk 2 above)
Comment (eriklsy) [reply frontend/components/ui/input-field.tsx]: Didn't clean it up properly oops

**Review by cameronl10 — APPROVED** [review]
lgtm

**Pull request merged**